src/srtp/srtp.c

```diff
@@ -315,6 +379,50 @@ int srtp_decrypt(struct srtp *srtp, struct mbuf *mb)
 		if (err)
 			return err;
 	}
+	else if (comp->aes && comp->mode == AES_MODE_GCM) {
+
```

Comment:
I suggest we replace the entire if block with this code for clarity, simplicity and consistence with RTCP code:
```
		union vect128 iv;
		uint8_t *p = mbuf_buf(mb);
		size_t tag_start;

		srtp_iv_calc_gcm(&iv, &comp->k_s, strm->ssrc, ix);

		aes_set_iv(comp->aes, iv.u8);

		/* The RTP Header is Associated Data */
		err = aes_decr(comp->aes, NULL, &mb->buf[start],
			       mb->pos - start);
		if (err)
			return err;

		if (mbuf_get_left(mb) < GCM_TAGLEN)
			return EBADMSG;

		tag_start = mb->end - GCM_TAGLEN;

		err = aes_decr(comp->aes, p, p, tag_start - mb->pos);
		if (err)
			return err;

		err = aes_authenticate(comp->aes, &mb->buf[tag_start],
				       GCM_TAGLEN);
		if (err)
			return err;

		mb->end = tag_start;

		/*
		 * 3.3.2.  Replay Protection
		 *
		 * Secure replay protection is only possible when
		 * integrity protection is present.
		 */
		if (!srtp_replay_check(&strm->replay_rtp, ix))
			return EALREADY;

```